recipes-ni/ni-systemformat/files/nisystemformat

```diff
@@ -648,15 +648,6 @@ if [ "$ENCRYPT" = yes ]; then
 		die_with_usage NO_TPM_HARDWARE \
 			"The TPM chip on this machine could not be detected (does it have one?)"
 	fi
-
-	# Sanity check, unlikely to hit
-	# Encrypted disk won't boot unless system is provisioned with initramfs enabled
```

Comment:
So have we never enabled encryption of configfs, rootfs partitions?

Reply:
Right, pretty sure its been broken for a while now,  The upstream API changed so most of the plumbing we have won't work even if we enabled it.